doc/source/release-notes/Percona-Server-5.7.10-2.rst

```diff
@@ -0,0 +1,80 @@
+.. rn:: 5.7.10-2
+
+===========================
+ |Percona Server| 5.7.10-2
+===========================
+
+Percona is glad to announce the second Release Candidate release of |Percona Server| 5.7.10-2 on February 4th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.10-2rc2/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.7.10 <http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-10.html>`_, including all the bug fixes in it, |Percona Server| 5.7.10-2 is the current Release Candidate release in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.7.10-2 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.7.10-2rc2>`_
+
+New Features
+============
+
+ Complete list of changes between |Percona Server| 5.6 and 5.7 can be seen in :ref:`changed_in_57`.
+
+ 5.7 binlog group commit algorithm is now supported in |TokuDB| as well.  
+
+Known Issues
+============
+
+ In |Percona Server| 5.7 `super_read_only <https://www.percona.com/doc/percona-server/5.6/management/super_read_only.html>`_ feature has been replaced with upstream implementation. There are currently two know issues compared to |Percona Server| 5.6 implementation:
+
+   * Bug :mysqlbug:`78963`, :variable:`super_read_only` aborts ``STOP SLAVE`` if variable :variable:`relay_log_info_repository` is set to ``TABLE`` which could lead to a server crash in Debug builds.
+
+   * Bug :mysqlbug:`79328`, :variable:`super_read_only` set as a server option has no effect.
+
+ Restarting the server with the :variable:`innodb_flush_method` set to ``ALL_O_DIRECT`` fails intermittently which could break crash recovery (bug :bug:`1529885`).
```

Comment:
InnoDB crash recovery might fail if innodb_flush_method is set to ALL_O_DIRECT. The workaround is to set this variable to a different value before starting up the crashed instance.